src/app/routes/legacyAsset/index.js

```diff
@@ -0,0 +1,8 @@
+import cpsAssetRoute from '../cpsAsset';
+import { legacyAssetPagePath } from '../regex';
+
+export default {
+  ...cpsAssetRoute,
+  path: legacyAssetPagePath,
+  pageType: 'legacyAsset',
+};
```

Comment:
> allow the path property to except an array of values (i.e. the 'old' and 'new' regexes - but this might require too much complexity)

We actually talked about doing this. We figured out where we would need to make the changes to enable this but then reasoned that since it is a different route, it should be a different route. Open to a discussion about this though as it would probably be the cleanest solution.

Reply:
Hi James, thanks for the feedback - I'll do the first 3 for now, to allow us to get Legacy assets on Test.  Yeah, having spoken with Jonathan yesterday, it feels like it would be good to allow multiple paths to map to a single page type, but it's a conversation we can have later.

Reply:
Cool thanks!